docs/en/docs-nav.json

```diff
@@ -277,6 +277,10 @@
                 {
                   "text": "Switch to PostgreSQL",
                   "path": "Entity-Framework-Core-PostgreSQL.md"
+                },
+				{
```

Comment:
Can you fix the indent. We don't use tabs, we use "two space" instead of a tab (for JSON files).

Reply:
OK, I've fixed: https://github.com/abpframework/abp/commit/525387654af80129f0836af8ca665b39c5e028ec